{"timestamp":1724027042807,"message":"Database error","log.level":"ERROR","logger.name":"api.blueprints.drbWork","thread.id":139783729318784,"thread.name":"MainThread","process.id":2354,"process.name":"MainProcess","file.name":"/home/runner/work/drb-etl-pipeline/drb-etl-pipeline/api/blueprints/drbWork.py","line.number":36,"entity.type":"SERVICE"}
